### ISSUE TITLE
Yapfify-buffer can't be called from after-save-hook.

### ISSUE THREAD

**Issue by proofit404** [JorisE/yapfify.el#6]
`yapf-mode` add `yapfify-buffer` to the `after-save-hook`. This hook calls function without argument in the non-interactive way. This cause arity error.

**Comment by JorisE** [comment]
Hi!  What do you mean by an arity error?

Also, yapf mode adds `yapfify-buffer` to `before-save-hook` but not `after-save-hook`.

**Comment by proofit404** [comment]
Arity error means function call doesn't match actual function signature.

Yes, I misspell hook name.

**Comment by proofit404** [comment]
This is error I get in my `*Messages*` buffer on every save with `yapf-mode` enabled.

```python
Saving file /home/proofit404/data/scoring/scoring/tele2/models.py...
Error: (wrong-number-of-arguments (lambda (beginning end) "Try to yapfify the current buffer.
If yapf exits with an error, the output will be shown in a help-window." (interactive "r") (let* ((original-buffer (current-buffer)) (original-point (point)) (original-window-pos (window-start)) (start-line (line-number-at-pos (if (use-region-p) beginning (point-min)))) (end-line (line-number-at-pos (if (use-region-p) (if (or (= (char-before end) 10) (= (char-before end) 13)) (- end 1) end) (point-max)))) (tmpbuf (generate-new-buffer "*yapfify*")) (exit-code (yapfify-call-bin original-buffer tmpbuf start-line end-line))) (deactivate-mark) (cond ((or (eq exit-code 0) (eq exit-code 2)) (save-current-buffer (set-buffer tmpbuf) (copy-to-buffer original-buffer (point-min) (point-max)))) ((eq exit-code 1) (error "Yapf failed, see *yapfify* buffer for details"))) (kill-buffer tmpbuf) (goto-char original-point) (set-window-start (selected-window) original-window-pos))) 0)
Wrote ‘/home/proofit404/data/scoring/scoring/tele2/models.py’
```

**Comment by toshism** [comment]
Ah, I broke this with my last pr. I'll submit a fix.

**Comment by JorisE** [comment]
This has been fixed in #7.
